EncoreTickets.SDK/Checkout/Models/RequestModels/DeliveryMethod.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace EncoreTickets.SDK.Checkout.Models.RequestModels
+{
+    /// <summary>
+    /// Delivery method used in the booking.
+    /// </summary>
+    public enum DeliveryMethod
+    {
+        /// <summary>
+        /// for print at home
+        /// </summary>
+        E,
```

Comment:
Is it possible to make this more readable (i. e. make the code convert readable enum names to whatever the service requires)?